src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeData.cxx

```diff
@@ -93,6 +94,10 @@ PoissonLogLikelihoodWithLinearModelForMeanAndListModeData<TargetT>::post_process
 
   if (this->list_mode_filename.length() == 0) 
   { warning("You need to specify an input file\n"); return true; } 
+
+  // handle time frame definitions etc
+    if(this->num_events_to_use==0 && this->frame_defs_filename.size() == 0)
+      do_time_frame = true;
```

Comment:
cannot be in `post_process()` as this is only called after parsing, while someone might have set these variables elsewhere (e.g. in Python)

Reply:
Then I guess set_up() is the place to put them ?